apps/ejabberd/src/mod_http_notification.erl

```diff
@@ -115,3 +89,14 @@ record_result(Host, {error, Reason}, _) ->
     mongoose_metrics:update([Host, mod_http_notifications, failed], 1),
     ?WARNING_MSG("Sending http notification failed: ~p", [Reason]),
     ok.
+
+%% @doc Convert to binary, strip initial slash (it is added by mongoose_http_client)
+fix_path(P) when is_list(P) ->
+    fix_path(list_to_binary(P));
+fix_path(P) when is_binary(P) ->
+    case P of
+        <<"/", R/binary>> ->
```

Comment:
There is no need for the `case` clause here, we can do the matching in the function head.